### ISSUE TITLE
Finetuning for emotion analysis but nan output

### ISSUE THREAD

**Issue by Changezi001** [zinengtang/TVLT#9]
Hi, 

I have downloaded the CMU-MOSEI dataset, and I am fine-tuning TVLT for emotion analysis on the MOSEI dataset. Although I could not obtain the labels_emotion folder in the downloaded zip file, I have compiled my own [emotion label file](https://github.com/Changezi001/CMU_MOSEI_emotion_labels) using the [timestamp file] (http://immortal.multicomp.cs.cmu.edu/timestamped_links/) from CMU, but when I use the [fine-tuning script][(url](https://github.com/zinengtang/TVLT/blob/main/DS.md)) to fine-tune it on CMU-MOSEI, the model outputs nan values after some iterations. Could you shed some light on what could be wrong?

**Comment by zinengtang** [comment]
Could you explain how did you convert the values to the labels?

**Comment by Changezi001** [comment]
I just took the [mosei.csv file](http://immortal.multicomp.cs.cmu.edu/timestamped_links/) and removed columns 1:5. But, before removing col 1:5, I changed column 0, i.e., id column, to FileName column by: column_0 + '_'  + column_2

**Comment by Changezi001** [comment]
And in [moseiemo_dataset.py](https://github.com/zinengtang/TVLT/blob/main/model/data/datasets/moseiemo_dataset.py), line number 78, I changed it to emolist = np.array(self.labels[self.labels['FileName']==self.keys[index]].iloc[0, 1:] > 0.0), and that gives me boolean labels
